cognite_toolkit/_cdf.py

```diff
@@ -381,8 +377,8 @@ def deploy(
             if result:
                 results[result.name] = result
             if ToolGlobals.failed:
-                print(f"[bold red]ERROR: [/] Failure to clean {LoaderCls.folder_name} as expected.")
-                exit(1)
+                raise ToolkitCleanResourceError(f"Failure to clean {loader_cls.folder_name} as expected.")
```

Comment:
This type of error has often turned out to b e difficult to debug. Is it a later task to raise it were `ToolGlobals.failed` is set to `True` instead of doing it here. Wdyt?